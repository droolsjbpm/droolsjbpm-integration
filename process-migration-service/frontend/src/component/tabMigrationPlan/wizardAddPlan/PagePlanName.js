import React, { Component } from "react";

export default class PagePlanName extends Component {
  componentDidMount() {
    if (document.getElementById("id_PagePlanName_name") != null) {
      document.getElementById("id_PagePlanName_name").value = this.props.name;
      document.getElementById(
        "id_PagePlanName_description"
      ).value = this.props.description;
    }
  }

  render() {
    return (
      <div className="form-horizontal">
        <div className="form-group required">
          <label className="col-sm-2 control-label">Name</label>
          <div className="col-sm-10">
            <input
              type="text"
              name="name"
              className="form-control"
              id="id_PagePlanName_name"
            />
          </div>
        </div>
        <div className="form-group">
          <label className="col-sm-2 control-label">Description</label>
          <div className="col-sm-10">
            <textarea
              name="description"
              rows="2"
              className="form-control"
              id="id_PagePlanName_description"
            />
          </div>
        </div>
      </div>
    );
  }
}
