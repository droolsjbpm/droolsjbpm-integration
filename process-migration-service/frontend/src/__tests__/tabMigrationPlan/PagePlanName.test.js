import renderer from "react-test-renderer";
import React from "react";
import PagePlanName from "../../component/tabMigrationPlan/wizardAddPlan/PagePlanName";

test("PagePlanName renders correctly using snapshot", () => {
  const tree = renderer
    .create(<PagePlanName name={"testName"} description={"testDescription"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
