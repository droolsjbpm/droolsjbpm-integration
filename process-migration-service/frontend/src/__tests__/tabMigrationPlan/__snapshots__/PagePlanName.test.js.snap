// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagePlanName renders correctly using snapshot 1`] = `
<div
  className="form-horizontal"
>
  <div
    className="form-group required"
  >
    <label
      className="col-sm-2 control-label"
    >
      Name
    </label>
    <div
      className="col-sm-10"
    >
      <input
        className="form-control"
        id="id_PagePlanName_name"
        name="name"
        type="text"
      />
    </div>
  </div>
  <div
    className="form-group"
  >
    <label
      className="col-sm-2 control-label"
    >
      Description
    </label>
    <div
      className="col-sm-10"
    >
      <textarea
        className="form-control"
        id="id_PagePlanName_description"
        name="description"
        rows="2"
      />
    </div>
  </div>
</div>
`;
